packages/uniforms-bridge-simple-schema-2/src/SimpleSchema2Bridge.ts

```diff
@@ -92,8 +92,7 @@ export default class SimpleSchema2Bridge extends Bridge {
 
   // eslint-disable-next-line complexity
   getProps(name: string, props: Record<string, any> = {}) {
-    // Type should be omitted.
-    // eslint-disable-next-line no-unused-vars, prefer-const
+    // eslint-disable-next-line prefer-const
     let { optional, type, uniforms, ...field } = this.getField(name);
 
     field = { ...field, required: !optional };
```

Comment:
We can easily avoid using `let` by naming `...field` differently
```suggestion
    const { optional, type, uniforms, ...contextField } = this.getField(name);

    const field = { ...contextField, required: !optional };
```
I know it's barely cosmetical change here, but I believe the `prefer-const` rule is there for a reason.